packages/dotcom-server-handlebars/src/helpers/renderJsxComponent.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+export default function renderJsxComponent({ hash }) {
+  let modulePath
+
+  if (hash.hasOwnProperty('package')) {
+    modulePath = path.join(process.cwd(), `node_modules/${hash.package}`)
+  }
+
+  // localRoute is relative to root of app consuming dotcom-server-handlebars.
+  if (hash.hasOwnProperty('localRoute')) {
+    modulePath = path.resolve(process.cwd(), `./${hash.localRoute}`)
```

Comment:
`path.resolve(hash.localRoute)` is the same =]

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/746/commits/87955298635169ad7f9ac628f062dfd91dd3d93e).